File: qualifier-week-3/non graded assignments/PPA4 - Not Graded.c
#include <stdio.h>

int main() {
    int n, sum = 0;
    scanf("%d", &n);

    for (int i = 0; i < n; i++) {
        sum += 2 * i + 1;
    }

    printf("%d\n", sum);

    return 0;
}

File: qualifier-week-3/non graded assignments/PPA4 - Not Graded.rs
fn main() {
    let mut n: i32 = 0;
    let mut sum = 0;

    println!("Enter a positive integer: ");
    let _ = scand!("{:i}", &mut n);

    for i in 0..n {
        sum += 2 * i + 1;
    }
    println!("Sum of the Series: {}", sum);
}